src/components/Event/Event.st.css

```diff
@@ -0,0 +1,143 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback, font;
+}
+
+:import {
+    -st-from: "../Text/Text.st.css";
+    -st-default: TPAText;
+}
+
+/*Defaults*/
+:vars {
+    DefaultFontStyle: "{theme: 'Body-M', size: '14px', lineHeight: '1em'}";
+    DefaultTimeColor: color-4;
+    DefaultTitleColor: color-5;
+    DefaultAllDayBackground: color-8;
+    DefaultAllDayOpacity: 0.2; 
+    DefaultTopPadding: 4px;
+    DefaultRightPadding: 4px;
+    DefaultBottomPadding: 4px;
+    DefaultLeftPadding: 12px;
+    DefaultRTLTopPadding: 4px;
+    DefaultRTLRightPadding: 12px;
+    DefaultRTLBottomPadding: 4px;
+    DefaultRTLLeftPadding: 4px;
+    BeforeColor: color(color-4);
+    BeforeOpacity: 0.06;
+}
+
+/*Overrides*/
+:vars {
+    FontStyle: --overridable;
+    TimeColor: --overridable;
+    TitleColor: --overridable;
+    AllDayBackground:  --overridable;
+    AllDayOpacity: --overridable;
+    TopPadding: --overridable;
+    RightPadding: --overridable;
+    BottomPadding: --overridable;
+    LeftPadding: --overridable;
+    RTLTopPadding: --overridable;
+    RTLRightPadding: --overridable;
+    RTLBottomPadding: --overridable;
+    RTLLeftPadding: --overridable;
```

Comment:
I'm not sure the paddings should be overridden
@michalvardi ?

Reply:
In any case, we also prefer css solutions to dealing with `rtl` ie `flex` and `::before` and `::after`

Reply:
This is not dealing with the `rtl`, it's just dealing the padding in the `rtl` case.

Reply:
It's kind of the same thing 😛 
But what I meant is that we prefer to avoid having specific solutions for `ltr` and `rtl`.

In addition, we use the `TPAComponentsConsumer` to pass the `rtl` state to stylable, when needed